addons/website/views/snippets/s_image_hexagonal.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_image_hexagonal" name="Image Hexagonal">
+    <section class="s_image_hexagonal o_cc o_cc1">
+        <div class="container-fluid">
+            <div class="row o_grid_mode" data-row-count="14">
+                <div class="o_grid_item g-col-lg-4 g-height-8 col-lg-4 pt24 pb24" style="grid-area: 3 / 2 / 12 / 6; z-index: 1;">
+                    <img
+                        src="/web_editor/image_shape/website.s_image_hexagonal_default_image_1/web_editor/geometric/geo_hexagon.svg" class="img img-fluid mx-auto" data-shape="web_editor/geometric/geo_hexagon" data-file-name="s_image_text.jpg" data-original-mimetype="image/jpeg" alt=""/>
+                </div>
+                <div class="o_grid_item g-col-lg-5 g-height-10 col-lg-4 pt24 pb24" style="grid-area: 3 / 5 / 13 / 9; z-index: 3;">
+                    <h2 class="display-4">Let’s discuss your project over a cup of coffee</h2>
+                    <p>Write one or two paragraphs describing your product or services. To be successful your content needs to be useful to your readers.</p>
+                    <p>Start with the customer – find out what they want and give it to them.</p>
+                    <p><a href="#" class="btn btn-primary">Contact Us</a></p>
```

Comment:
Suggestion 
```suggestion
                    <p>Start with the customer – find out what they want and give it to them.</p>
                    <p><a href="#" class="btn btn-secondary btn-lg">Contact Us</a></p>
```

| this PR | proposal |
|--------|--------|
| ![image](https://github.com/user-attachments/assets/55cde72b-df7d-42a6-b20f-d914c932f0b8) | ![image](https://github.com/user-attachments/assets/b0bd8340-15fc-4003-831e-2c25b863e53e) |